packages/commonwealth/client/scripts/views/pages/Community/Groups/Update/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import app from 'state';
+// import { useEditGroupMutation } from 'state/api/groups';
+import Permissions from 'utils/Permissions';
+import { PageNotFound } from '../../../404';
+import { GroupForm } from '../common/GroupForm';
+import './index.scss';
+
+const UpdateCommunityGroupPage = () => {
+  // const { mutateAsync: editGroup } = useEditGroupMutation();
+
+  if (!app.isLoggedIn() || !Permissions.isCommunityAdmin()) {
+    return <PageNotFound />;
+  }
+
+  // TODO: we will get these values from existing group data
+  const initialValues = {
+    groupName: 'Ninja Turtles',
+    groupDescription: 'This is a group for turtles that became a ninja',
+    requirements: [
+      {
+        requirementAmount: '11',
+        requirementChain: {
+          label: 'Ethereum',
+          value: 'Ethereum',
+        },
+        requirementCondition: {
+          label: 'More than',
+          value: 'More than',
+        },
+        requirementContractAddress: '1234567ugcvy',
+        requirementType: {
+          label: 'ERC-20',
+          value: 'ERC-20',
+        },
+      },
+    ],
+    requirementsToFulfill: 5 as any,
+    topics: [
+      {
+        label: 'Change Log',
+        value: 'Change Log',
+      },
+      {
+        label: 'Communities',
+        value: 'Communities',
+      },
+    ],
+  };
+
+  return (
+    <GroupForm
+      formType="edit"
+      initialValues={initialValues}
```

Comment:
This prop will fill the form fields with the values in the `initialValues` object. Very helpful when the form is used to update existing data (which we can pre-fill in the form very easily)